src/docker-manager.js

```diff
@@ -83,16 +84,29 @@ module.exports = class DockerManager {
         this.imageName = `${image}:${version}`;
         log.debug(`=> Pulling ${this.imageName}`);
         return new Promise((resolve, reject) => {
+            spinner.update('Pulling docker image');
```

Comment:
GUI updates should not be handled inside the `docker-manager.js` file. Can you move all these references into `index.js`? That is why I have added `rxjs` a while back ago to have constant updates on the progress indication, since promises only resolve or reject.

Reply:
Will do!

Reply:
I moved all of this code to `index.js`. Tell me if you like it!